modelstore/storage/hosted.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         self.secret_access_key = _get_environ(
             secret_access_key, "MODELSTORE_ACCESS_KEY"
         )
+        self.root_dir = ""
```

Comment:
The `HostedStorage` is an experimental API-based storage, which doesn't expose specific paths to models. So this one is not required 🔪  (I can clean this up separately if you want)

Reply:
Yeah, I was adding this because the path.py needs it. But putting it in the base class would be a good idea.

Reply:
Fixed in https://github.com/operatorai/modelstore/pull/116